README.md

```diff
@@ -3,3 +3,13 @@ A plugin for rendering tiled maps. Specfically maps from the tiled editor which
 https://www.mapeditor.org/
 
 Feel free to use this code as a reference for your own custom tile mapping solution as well.
+
+## examples
+
+Follow the [Setup guide](https://bevyengine.org/learn/book/getting-started/setup/) to ensure your development environment is set up correctly.
+Once set up, you can quickly try out the [examples](/examples) by cloning this repo and running the following command:
+
+```sh
+# Runs the "breakout" example
+cargo run --example breakout
+```
```

Comment:
Could you do something like: 
```suggestion

```sh
# Runs the orthographic tile map example
cargo run --example ortho_main
```
```